packages-ts/starknet-gauntlet-starkgate/src/commands/L1-bridge/Evm-artifacts/L1Bridge.abi

```diff
@@ -0,0 +1,485 @@
+[
```

Comment:
Should these artifacts be distributed with npm? @krebernisak

Reply:
@sdrug  can we remove the abis? I'm assuming no bc I think it's also included in https://github.com/smartcontractkit/chainlink-starknet/pull/73

Reply:
Yes, we shouldn't include any contract artifacts in the gauntlet modules.
In fact, those were gone already in the bridge PR.